packages/manager/apps/dedicated/client/app/components/dedicated-cloud/taskPollProgress/taskPollProgress.html

```diff
@@ -0,0 +1,36 @@
+<span data-translate="dedicatedCloud_task_progress_operation"> </span>
+<span
+    class="ml-1"
+    data-translate="{{ 'dedicatedCloud_task_progress_operation_' + $ctrl.task.name }}"
+>
+</span>
+<span
+    class="ml-2"
+    data-translate="dedicatedCloud_task_progress_task_reference"
+    data-translate-values="{ task: $ctrl.task.taskId }"
+>
+</span>
+<div class="progress">
```

Comment:
Why not use https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-progress-introduction--page ?

Reply:
I can't use because the data-type is bind one time while i need change type linked with status of task.